src/test/java/scenarios/PaymentsAndRefundTest.java

```diff
@@ -248,7 +248,6 @@ public void verifyPaymentByMultipleModeThroughStoreManager(){
         float updateAmount1 = Float.parseFloat(updatedAmt1.replace(",", ""));
```

Comment:
Avoid using magic numbers or directly referencing indices (e.g., "1", ","). Create constants or provide meaningful names to these values.